File: Python/day14.py
num = int(input("Enter number to be reversed: "))
rev = 0
while num > 0:
    dig= num % 10  
    rev = rev * 10 + dig
    num = num // 10  
print("Reversed number is:", rev)


# num = input("Enter number to be reversed: ")
# rev = num[::-1] 
# print("Reversed number is:", rev)

File: Java/day14.java
import java.util.Scanner;

public class ReverseNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter number to be reversed: ");
        int num = scanner.nextInt();
        int rev = 0;
        while (num != 0) {
            int digit = num % 10;  
            rev = rev * 10 + digit;  
            num /= 10; 
        }
        System.out.println("Reversed number is: " + rev);
    }
}
